src/Python/interactive.py

```diff
@@ -166,6 +165,43 @@ def main():
 
     DEFAULT = input_("Enter the default command:")
 
+    # license work
+    def load_license_files(license_directory):
+        licenses = []
+        for filename in os.listdir(license_directory):
+            if filename.endswith(".txt"):
+                with open(os.path.join(license_directory, filename), 'rb') as file:
+                    licenses.append((filename[:-4], file.read().decode('utf-8'), os.path.join(license_directory, filename)))
+        return licenses
+
+    # license_directory = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../licenses")
+    script_directory = os.path.dirname(os.path.abspath(__file__))
+    license_directory = os.path.join(script_directory, "licenses")
+    license_list = load_license_files(license_directory)
+
+    def license_yes_no(prompt):
+        response = input(prompt + " [y/n] ").strip().lower()
+        return response == 'y' or response == 'yes'
+
+    include_license = license_yes_no("Would you like to include a license?")
+
+    if include_license:
+        print("Available licenses: ")
+        for index, (license_name, _, _) in enumerate(license_list, start=1):
+            print(f"{index}. {license_name}")
+
+        selection = input_(">>> Please select a license by entering its number: ")
+
+        while not selection.isdigit() or not 1 <= int(selection) <= len(license_list):
+            print("INVALID selection. Please provide the input again...\n")
+            selection = input_(">>> Please select a license by entering its number: ")
+
+        selected_license_name, selected_license_text, _ = license_list[int(selection) - 1]
+
```

Comment:
first copy license to user account as LICENSE.txt
then add to contents.. ("LICENSE.txt", "LICENSE")